manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: RakutenSymphonyKorea.SendAnywhere
+PackageVersion: 24.3.80853
+PackageLocale: en-US
+Publisher: Rakuten Symphony Korea, Inc.

```

```suggestion
Publisher: Estmob Inc.
```
This must match control panel